### PR TITLE
Fix the JSON style to camelCase and rename `CONTRIBUTE.md` to `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by mertyildiran**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
 ![Mizu: The API Traffic Viewer for Kubernetes](assets/mizu-logo.svg)
-# CONTRIBUTE
+
+# Contributing to Mizu
```
Comment (IgorGov) [inline CONTRIBUTING.md:3]: Can we move this file to .github folder?

File: ui/src/components/EntriesList.tsx [hunk 2]
```diff
@@ -54,8 +54,8 @@ export const EntriesList: React.FC<EntriesListProps> = ({entries, setEntries, fo
     const filterEntries = useCallback((entry) => {
         if(methodsFilter.length > 0 && !methodsFilter.includes(entry.method.toLowerCase())) return;
         if(pathFilter && entry.path?.toLowerCase()?.indexOf(pathFilter) === -1) return;
-        if(statusFilter.includes(StatusType.SUCCESS) && entry.status_code >= 400) return;
-        if(statusFilter.includes(StatusType.ERROR) && entry.status_code < 400) return;
+        if(statusFilter.includes(StatusType.SUCCESS) && entry.statusCode >= 400) return;
```
Comment (IgorGov) [inline ui/src/components/EntriesList.tsx:57]: Comment to myself: why code owners didn't work here?

File: CONTRIBUTING.md (hunk 1 above)
Comment (mertyildiran) [reply CONTRIBUTING.md:3]: @IgorGov the right place for this file is the root directory as far as I know. See the GitHub's related doc page https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Comment (mertyildiran) [reply CONTRIBUTING.md]: @IgorGov similarly moved it to `.github/` directory.

**Review by gadotroee — APPROVED** [review]

**Pull request merged**